kinds/librarypanel/librarypanel_kind.cue

```diff
@@ -0,0 +1,63 @@
+package kind
+
+import "strings"
+
+name:     "LibraryPanel"
+maturity: "experimental"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// 0.0
+			{
+				@grafana(TSVeneer="type")
+
+				// Folder UID
+				folderUid?: string @grafanamaturity(ToMetadata="sys")
+
+				// Library element UID
+				uid: string
+
+				// Panel name (also saved in the model)
+				name: string & strings.MinRunes(1)
```

Comment:
What does `& strings.MinRunes(1)` do? I don't see any difference in either go or TS output.

Reply:
copied from dashboard... i means it has to be a non-empty string

(that constraint is not visible in typescript!)